CHANGELOG/CHANGELOG-1.4.md

```diff
@@ -0,0 +1,134 @@
+# Changelog
+
+## [Unreleased](https://github.com/diem/diem/tree/HEAD)
+
+[Full Changelog](https://github.com/diem/diem/compare/diem-core-v1.3.0...HEAD)
+
+**Notable changes:**
+
+**[Consensus]**
+
+- Added decoupled execution. See related PRs ([\#8616](https://github.com/diem/diem/pull/8616), [\#8626](https://github.com/diem/diem/pull/8626), [\#8743](https://github.com/diem/diem/pull/8743), [\#8754](https://github.com/diem/diem/pull/8754), [\#8774](https://github.com/diem/diem/pull/8774), [\#8784](https://github.com/diem/diem/pull/8784), [\#8701](https://github.com/diem/diem/pull/8701), [\#8806](https://github.com/diem/diem/pull/8806), [\#8792](https://github.com/diem/diem/pull/8792), [\#8653](https://github.com/diem/diem/pull/8653))
+- \[decoupled-execution\] Add a new API for the TSafetyRules trait [\#8657](https://github.com/diem/diem/pull/8657)
+
+**[Diem Framework]**
+
+- Rename DiemID move resources and rust structures to VASPDomain [\#8695](https://github.com/diem/diem/pull/8695)
+- \[read/write set analysis\] Fix join child data [\#8751](https://github.com/diem/diem/pull/8751)
```

Comment:
I think we should omit these 3 PRs for the read/write set analysis. For now, this is an experimental feature under development to support parallel execution in the future.